examples/carbon-for-ibm-products/AboutModal/src/Example/Example.jsx

```diff
@@ -82,7 +82,7 @@ export const Example = () => {
             IBM{' '}
             <span
               style={
-                // stylelint-disable-next-line carbon/type-token-use
+                
```

Comment:
```suggestion
```
Probably don’t mean to keep this blank line here?

Reply:
yes unfortunately this added a bunch of new lines i'll have to manually remove because `prettier` won't haha